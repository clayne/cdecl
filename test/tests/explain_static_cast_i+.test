cdecl @ /dev/null @ -xc++ @ explain static_cast<int>(S::x) @ 0
