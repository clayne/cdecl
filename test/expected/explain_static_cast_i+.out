static cast x of scope S into int
